src/main/java/seedu/contax/logic/commands/EditAppointmentCommand.java

```diff
@@ -93,6 +93,8 @@ public CommandResult execute(Model model) throws CommandException {
         Appointment editedAppointment = createEditedAppointment(appointmentToEdit,
                 model.getFilteredPersonList(), editAppointmentDescriptor);
 
+        editedAppointment = editedAppointment.setPriority(appointmentToEdit.getPriority());
+
```

Comment:
What is this for?

Reply:
Get the original priority

Reply:
But in your editappointment need to accept the original priority, either need to add here or EditAppointmentCommand#createEditedAppointment()